addons/website/static/src/scss/website.ui.scss

```diff
@@ -147,3 +147,7 @@ body {
         scrollbar-width: thin;
     }
 }
+
+body.modal-open:has(.js_sale.o_wsale_product_page) {
```

Comment:
This style only targets the product page, so it should probably be moved to `website_sale` :)